onnx/test/cpp/function_context_test.cc

```diff
@@ -91,7 +107,24 @@ void RegisterTwiceFloatSchema() {
   (void)unused;
 }
 
+void RegisterTwiceSchema() {
+  ONNX_NAMESPACE::OpSchema schema;
+  schema.SetName("CustomFunTwice")
+      .SetDomain(ONNX_DOMAIN)
+      .SinceVersion(12)
+      .SetDoc("This operator returns an output tensor that is twice the input tensor.")
+      .Input(0, "X", "Input tensor", "T", OpSchema::Single)
+      .Output(0, "Y", "Output tensor", "T", OpSchema::Single)
+      .TypeConstraint("T", {"tensor(float)", "tensor(double)"}, "Type of the input and output values")
+      .SetContextDependentFunctionBodyBuilder(BuildTwiceFunctionBody);
+#ifndef ONNX_NO_EXCEPTIONS
+  ONNX_NAMESPACE::OpSchemaRegistry::OpSchemaRegisterOnce unused(schema);
+  (void)unused;
+#endif
+}
```

Comment:
Why doesn't RegisterTwiceFloatSchema need a similar update?

Reply:
This test was not written properly - RegisterTwiceFloatSchema would register the CustomFunc for the first time and then it was registered for the second time here. Anyways I updated both the test to remove a lot of common code and make the intention of the tests clear.